### PR TITLE
Sequence text in a tile spans tile-width

### PR DESCRIPTION
**Pull request by joswinter**
Fixes Sequence text in a tile in a graph viewer does not span the tile-width.

### CODE REVIEW

File: src/main/java/nl/tudelft/lifetiles/graph/view/VertexView.java [hunk 1]
```diff
@@ -77,26 +73,25 @@
     public VertexView(final String string, final double initX,
             final double initY, final double width, final double height,
             final Color color) {
-        this.text = new Text(string);
+        text = new Text(string);
         text.setTextOrigin(VPos.CENTER);
         text.setFill(Color.WHITE);
         text.setFontSmoothingType(FontSmoothingType.LCD);
-        text.setFont(Font.font("Open Sans", FONTSIZE));
+        text.setFont(Font.font("Oxygen Mono", HORIZONTALSCALE));
```
Comment (jorenham) [inline src/main/java/nl/tudelft/lifetiles/graph/view/VertexView.java:79]: The font name should be a `static final` constant.

**Comment by jorenham** [comment]
Might be cleaner to push the font files directly to `dev`

**Comment by albert-smit** [comment]
looks good, but i agree that the font name should be a constant

File: src/main/java/nl/tudelft/lifetiles/graph/view/VertexView.java [hunk 2]
```diff
@@ -122,17 +117,16 @@ public final double getWidth() {
...
+        double fontWidth = text.getLayoutBounds().getWidth();
+        text.setFont(Font.font("Oxygen Mono", (HORIZONTALSCALE) * width
+                / fontWidth));
+
         clip.setWidth(width);
         clip.setHeight(height);
         clip.setLayoutX(0);
         clip.setLayoutY(-height / 2);
 
-        if (resizeWidth == -1) {
```
Comment (alangerak) [inline src/main/java/nl/tudelft/lifetiles/graph/view/VertexView.java:130]: resizeWidth variable is unused now, should be removed

**Comment by joswinter** [comment]
Added static final to VertexView and Main, also removed resizeWidth

**Pull request merged**